CHANGES.rst

```diff
@@ -9,7 +9,12 @@ General
 Bug Fixes
 ---------
 
--
+  
+ramp_fitting
+~~~~~~~~~~~~
+
+- Fixed zeros that should be NaNs in rateints product and suppressed
```

Comment:
Minor nit, but doesn't this change affect both rate and rateints products? Or was the result in the rate product already being set to NaN and the problem was just in the results for individual integrations?

Reply:
It affects both.  I updated the change log.